spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'bundler/setup'
 require 'cocina/models'
+require 'cocina/rspec'
```

Comment:
This allows using the matchers in this app, which allows testing the above change to the matcher.